tests/lib/rules/jsx-first-prop-new-line.js

```diff
@@ -91,6 +91,39 @@ ruleTester.run('jsx-first-prop-new-line', rule, {
       parser: parserOptions
     },
     {
+      code: [
+        '<Foo bar />'
+      ].join('\n'),
+      options: ['multiline-multiprop'],
+      parser: parserOptions
+    },
+    {
+      code: [
+        '<Foo bar baz />'
+      ].join('\n'),
+      options: ['multiline-multiprop'],
+      parser: parserOptions
+    },
+    {
+      code: [
+        '<Foo prop={{',
```

Comment:
similarly, this should be invalid.

just like an object literal, or an array literal, if 100% of the expression can't fit on one line, then every item needs to appear on a line by itself.